### DISCUSSION TITLE
Showing group in filter but cant set

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#13298] [Archive]
Filter has a and/or group option but is not able to set group option like and/or. is it disabled or yet to build?

i through we select group then aggregate sum, count, countDistinct will be available to configure

![image](https://user-images.githubusercontent.com/36895131/168409335-e3a32f6d-4f90-4a00-8988-2b0d9f9ce881.png)

### To Reproduce

try to click group option

### Errors Shown

not able to select group

**Comment by azrikahar** [comment]
I believe there might be a misunderstanding here. "And/Or group" in this context is different to the [group](https://docs.directus.io/reference/query/#aggregation-grouping) in global query parameters. 

It's only meant as a "group" of filters (or nested and/or) within it. The `OR` group in your filter shown in the screenshot means any filters nested inside it will be evaluated in an OR logic rather than the default AND. Here's an example of the nesting I am referring to:

![chrome_RLYFNjcDYk](https://user-images.githubusercontent.com/42867097/168449737-3fe6c495-096f-4cd3-9126-296f9e980c0d.png)

With that said, #12488 has actually added support for functions in `9.8.0` so that opens up a lot more possibilities in terms of filtering! As for aggregation such as `average`, `sum` etc, you can upvote and join in on the discussion over in #11239 :thumbsup:
